src/app.js

```diff
@@ -1,3 +1,13 @@
-const app = async () => '#BuildforSDG';
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Header from './components/Header/Header';
 
```

Comment:
Welldone @Yasholma , nice work, i was able to run it successfully, though i needed to change the name of this file from `app.js` to `App.js` because i was getting an error  while compiling, due to the App component being imported in `index.js` by referencing `./App.js`.

I will recommend you rename this file to `App.js` or what do you think?

Reply:
Yea, I was supposed to delete that test until we start something useful. My bad I missed that.